memorystore/redis/cloud_run_deployment/e2e_test.py

```diff
@@ -24,83 +24,42 @@
 
 import pytest
 
+# Unique suffix to create distinct service names
 
-@pytest.fixture()
-def services():
-    # Unique suffix to create distinct service names
-    suffix = uuid.uuid4().hex[:10]
-    project = os.environ["GOOGLE_CLOUD_PROJECT"]
+SUFFIX = uuid.uuid4().hex[:10]
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+VPC_NETWORK_NAME = "memorystore-redis"
```

Comment:
Since the test isn't creating a VPC connector anymore, it doesn't look like we need this variable

Reply:
Good eyes! I'll make that fix.